x-pack/plugins/cases/server/client/cases/find.ts

```diff
@@ -100,20 +94,7 @@ export const find = async ({
       subCaseOptions: caseQueries.subCase,
     });
 
-    for (const theCase of cases.casesMap.values()) {
-      try {
-        ensureSavedObjectIsAuthorized(theCase.owner);
-        // log each of the found cases
-        auditLogger?.log(
-          createAuditMsg({ operation: Operations.findCases, savedObjectID: theCase.id })
-        );
-      } catch (error) {
-        auditLogger?.log(
-          createAuditMsg({ operation: Operations.findCases, error, savedObjectID: theCase.id })
-        );
-        throw error;
-      }
-    }
+    ensureAuthorizedForSavedObjects([...cases.casesMap.values()]);
```

Comment:
This now takes an array of owners to ensure that we have authorization for them.